content/en/docs/concepts/configuration/secret.md

```diff
@@ -43,21 +53,25 @@ The name of a Secret object must be a valid
 The keys of `data` and `stringData` must consist of alphanumeric characters,
 `-`, `_` or `.`.
 
+
 ### Built-in Secrets
```

Comment:
What are built-in secrets?
I suggest you add some brief content below this heading before beginning with the sub-headings.

Reply:
Right. This is strange.

Reply:
Nice spot